src/app.ts

```diff
@@ -0,0 +1,21 @@
+// This was originally wrapped in a do block in
+// coffeescript, and it's not clear now why that was the
+// case, so I'm going to maintain that behaviour
```

Comment:
I also don't see any reason. Suggest dropping the closure.

Reply:
agreed, after a conversation with @Page-